packages/contracts-bedrock/src/universal/CrossDomainMessenger.sol

```diff
@@ -183,7 +179,7 @@ abstract contract CrossDomainMessenger is
         // guarantee the property that the call to the target contract will always have at least
         // the minimum gas limit specified by the user.
         _sendMessage(
```

Comment:
It would be amazing if you could update the calls in this contract to use the kwargs style syntax where its

```solidity
_sendMessage({
  key: value,
  ...
})
```

Reply:
This will help to reduce cognitive overhead during review. Prefer this style anytime there are 3 or more arguments or its not obvious what the arguments are